scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -495,6 +500,103 @@ object TableRowJsonIO {
   )
 }
 
+final case class BigQueryDynamicTable[T: Coder](writer: beam.BigQueryIO.Write[T])(
```

Comment:
Here since we are keeping this IO for now, we can habe `DynamicDestinations` as the only argument here.